src/context/AuthContext.tsx

```
@@ -0,0 +1,91 @@
+
+import React, { createContext, useState, useContext, useEffect } from 'react';
+import api from '@/lib/api';
+
+interface AuthContextType {
+  isAuthenticated: boolean;
+  token: string | null;
+  login: (data: any) => Promise<void>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)